llvm/test/CodeGen/SPIRV/hlsl-intrinsics/SV_GroupIndex.ll

```diff
@@ -0,0 +1,32 @@
+; RUN: llc -O0 -verify-machineinstrs -mtriple=spirv-vulkan-unknown %s -o - | FileCheck %s
+; RUN: %if spirv-tools %{ llc -O0 -mtriple=spirv-vulkan-unknown %s -o - -filetype=obj | spirv-val %}
+
+; CHECK-DAG:        %[[#int:]] = OpTypeInt 32 0
+; CHECK-DAG:        %[[#ptr_Input_int:]] = OpTypePointer Input %[[#int]]
+; CHECK-DAG:        %[[#LocalInvocationIndex:]] = OpVariable %[[#ptr_Input_int]] Input
+
+; CHECK-DAG:        OpEntryPoint GLCompute {{.*}} %[[#LocalInvocationIndex]]
+; CHECK-DAG:        OpName %[[#LocalInvocationIndex]] "__spirv_BuiltInLocalInvocationIndex"
+; CHECK-DAG:        OpDecorate %[[#LocalInvocationIndex]] LinkageAttributes "__spirv_BuiltInLocalInvocationIndex" Import
```

Comment:
We don't want the input variable to have a linkage attribute. This is not valid for SPIR-V shaders.

Reply:
This decoration is currently being added for `SV_GroupThreadID` and `SV_DispatchThreadID` as well. Should I change what is being generated for those as well, or simply remove the test for now?

Reply:
I'd say either remove this check, and create an todo/issue to track this cleanup, or change what's being generated.
But IMO adding a `CHECK` line for this would mean `this is the desired behavior`

Reply:
Okay, I've created an issue for this (#131878) and I'll remove the `CHECK`.